cmd/geth/main.go

```diff
@@ -261,6 +262,10 @@ func initGenesis(ctx *cli.Context) error {
 		utils.Fatalf("must supply path to genesis JSON file")
 	}
 
+	if ctx.GlobalBool(utils.TestNetFlag.Name) {
+		state.StartingNonce = 1048576 // (2**20)
+	}
```

Comment:
Ah, I see, you mentioned the issue in https://github.com/ethereum/go-ethereum/issues/2905#issuecomment-240760307